src/azure-cli/azure/cli/command_modules/botservice/webappv4.template.json

```diff
@@ -148,7 +156,7 @@
             ]
         },
         {
-            "apiVersion": "2017-12-01",
+            "apiVersion": "2018-07-12",
```

Comment:
I don't see a test for the `az bot create -k webapp` flow with the updates in the ARM template. This `apiVersion` doesn't match with when the KeyVault support was added, which was in 2020. (right @RustyDes?)

Reply:
This came from a sample Rusty provided. @RustyDes could you please confirm?

Reply:
If @RustyDes signed off, then it works for me 👍🏼

Reply:
@swagatmishra2007 I got the version from you so you should be the one signing off on that :)

Reply:
version should be 2020-06-02